pages/implementation/avo-inspector-sdk-reference.mdx

```diff
@@ -0,0 +1,54 @@
+---
+layout: 'guide'
+title: 'Avo Inspector SDK reference'
+---
+
+import Link from '../../components/Link';
+
+**Avo Inspector** analyzes the analytics events sent from your app, without collecting the actual data (not yet another tool to add to your privacy policy). Based on that analysis Avo automatically builds a single source of truth tracking plan, covering all your products and platforms, giving you a complete overview of which is tracking what, and which is not.
```

Comment:
Link to the getting started guides here?

Reply:
I think everything that's in the guide is covered here 🤔